kastenwesen.py

```diff
@@ -645,16 +645,23 @@ def needs_package_updates(self):
         # the temporary label is set so that check_for_unmanaged_containers()
         # does not complain about this "unmanaged" instance
 
+        def strip_port_args(docker_options):
+            """Return docker_options without ports to bind."""
+            return re.sub(
+                r'\s*-(-?)p(ublish)?[\s=]*\d[\d:\-\.]+\s*', ' ', docker_options,
+            )
+
         cmd = "docker run --rm" \
             " --dns-search=." \
             " --label de.fau.fablab.kastenwesen.temporary=True" \
             " --user=root" \
             " -v {kastenwesen_path}/helper/:/usr/local/kastenwesen_tmp/:ro{vol_opts}" \
-            " --name={new_name}" \
```

Comment:
`wp-cli` for wordpress needs to connect to the database. Therefore it needs to use the docker links, which are part of `docker_options`. Ok the MAC address will break things too... Should we strip out `--mac-address` too or is this too dirty?